aiohttp/web_urldispatcher.py

```diff
@@ -977,9 +977,13 @@ def register_resource(self, resource: AbstractResource) -> None:
         name = resource.name
 
         if name is not None:
+            if keyword.iskeyword(name):
+                raise ValueError('Incorrect route name {!r}, '
+                                 'python keywords cannot be used '
+                                 'for route name'.format(name))
```

Comment:
FYI this could be an f-string
```suggestion
                raise ValueError(f'Incorrect route name {name!r}, '
                                 'python keywords cannot be used '
                                 'for route name')
```

Reply:
Python 3.5 doesn't support f-strings.